### PR TITLE
add bein sports hd for Qatar and UAE

### PR DESCRIPTION
**Pull request by jlapitan**

### CODE REVIEW

File: channels/kr.m3u [hunk 1]
```diff
@@ -165,3 +165,5 @@ http://54.180.143.6:9981/stream/channelid/750850617
 http://54.180.143.6:9981/stream/channelid/1805607194
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/r1WTYTa.png" group-title="",현대홈쇼핑+샵 / + Shop
 http://54.180.143.6:9981/stream/channelid/965307250
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/ubZWt4D.png" group-title="",Animax
+http://1.244.176.6:9981/stream/channelid/2018783345
```
Comment (freearhey) [inline channels/kr.m3u]: stream does not work

File: channels/ae.m3u [hunk 2]
```diff
@@ -177,3 +177,5 @@ http://livecdnh1.tvanywhere.ae:80/hls/star_world/05.m3u8
 http://51.158.73.206/hls/stream.m3u8
 #EXTINF:-1 tvg-id="Wanasah ARB" tvg-name="Wanasah ARB" tvg-logo="https://i.imgur.com/GmgibYW.jpg" group-title="",Wanasah
 http://livecdnh2.tvanywhere.ae/hls/wanasah/04.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```
Comment (freearhey) [inline channels/ae.m3u]: we can’t add this channel to this playlist because it’s the Qatar channel

**Review by freearhey — CHANGES_REQUESTED** [review]
Please make changes so we can approve this PR.

File: channels/ae.m3u (hunk 2 above)
Comment (jlapitan) [reply channels/ae.m3u]: latest changes done

File: channels/qa.m3u [hunk 3]
```diff
@@ -17,3 +17,6 @@ https://alrayyan1live.azureedge.net/alrayyan1/alrayyan1.smil/chunklist_b1428000.
 https://alrayyan2live.azureedge.net/alrayyan2/alrayyan2.smil/chunklist_b1428000.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
 http://95.170.215.101/hls/m3u8/Bein-Sport-HD1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```
Comment (boomski) [inline channels/qa.m3u]: Right now this link does not work anymore
Most of the time these type of links are testlinks from a paid service and will work 24 or 48 Hours.

**Pull request closed without merging**